github/ci/prow/files/jobs/kubevirt/kubectl-virt-plugin/kubectl-virt-plugin-periodics.yaml

```diff
@@ -0,0 +1,49 @@
+periodics:
+- name: periodic-kubectl-virt-plugin-publish
+  cron: "0 0 * * *"
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  extra_refs:
+  - org: kubevirt
+    repo: kubectl-virt-plugin
+    base_ref: master
+    work_dir: true
+  - org: kubevirt
+    repo: project-infra
+    base_ref: master
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror: "true"
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: quay.io/kubevirtci/kubectl-virt-builder@sha256:554db95bb081e1ca404fc22cb140e868df9231b7d5578a74ead1bbda0f151545
```

Comment:
It would be great to make this image proxy-friendly, like we did here https://github.com/kubevirt/project-infra/pull/750 with kubevirt-infra-bootstrap, just changing the `FROM` to `ubevirtci/bootstrap:v20201119-a5880e0` should be enough. I can propose a PR, where is it built?

Reply:
The images are being built from [this repository](https://github.com/kubevirt/kubectl-virt-plugin/tree/master/images). Not sure, should we rather move them to project-infra?

Reply:
Awesome thx, PR proposed here https://github.com/kubevirt/kubectl-virt-plugin/pull/28

> should we rather move them to project-infra?

Not sure either, as I see it it makes sense to keep the image build process together with the rest of kubectl-virt-plugin related code as it is now, wdyt?

Reply:
OK, then let's keep them there :)